awswrangler/data_api/redshift.py

```diff
@@ -52,15 +56,33 @@ def __init__(
         retries: int = 15,
         boto3_session: Optional[boto3.Session] = None,
     ) -> None:
+        super().__init__()
+
+        self.client = _utils.client(service_name="redshift-data", session=boto3_session)
+        self.logger = logging.getLogger(__name__)
```

Comment:
Any reason why we set the logger as a class variable here but at the module level in `rds.py`?

Reply:
I moved it in `rds.py` but forgot to do the same here :)